content/reference/cli/supported-commands.md

```diff
@@ -0,0 +1,99 @@
+---
+title: "Cli Commands"
+---
+
+# Cli Commands
+
+## Setup and Config
+
+| Component | Supported  | Notes and limitations                   |
+|:----------|:-----------|:----------------------------------------|
+| `config`  | ✅          | planned support for `dolt config alias` |
+
+## Getting and Creating Projects
+
+| Component | Supported  | Notes and limitations |
+|:----------|:-----------|:----------------------|
+| `init`    | ✅          |                       |
+| `clone`   | ✅          |                       |
+
+## Basic Snapshotting
+
+| Component | Supported | Notes and limitations |
+|:----------|:----------|:----------------------|
+| `add`     | ✅         |                       |
+| `status`  | ✅         |                       |
+| `commit`  | ✅         |                       |
+| `notes`   | ❌         |                       |
+| `restore` | ❌         |                       |
+| `reset`   | ✅         |                       |
+| `rm`      | ✅         |                       |
+
+## Branching and Merging
+
+| Component  | Supported | Notes and limitations                                       |
+|:-----------|:----------|:------------------------------------------------------------|
+| `branch`   | ✅         |                                                             |
+| `checkout` | ✅         |                                                             |
+| `merge`    | ✅         |                                                             |
+| `log`      | 🟠        | planned support for `dolt log --all` and `dolt log --graph` |
+| `stash`    | ❌         |                                                             |
+| `tag`      | ✅         |                                                             |
+| `worktree` | ❌         |                                                             |
+
+## Sharing and Updating Projects
+
+| Component | Supported | Notes and limitations |
+|:----------|:----------|:----------------------|
+| `fetch`   | ✅         |                       |
+| `pull`    | ✅         |                       |
+| `push`    | ✅         |                       |
+| `remote`  | ✅         |                       |
+
+## Inspection and Comparison
+
+| Component    | Supported | Notes and limitations                               |
+|:-------------|:----------|:----------------------------------------------------|
+| `show`       | 🟠        | not supported for merge commits or internal objects |
+| `diff`       | ✅         |                                                     |
+| `range-diff` | ❌         |                                                     |
+| `shortlog`   | ❌         |                                                     |
+| `describe`   | ❌         |                                                     |
```

Comment:
describe should probably be dropped from the list. I don't think it has meaning in Dolt. I'm surprised it's listed so highly in the git docs for that matter - it pretty obscure and not useful to most people.